src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/MinMaxTable.cs

```diff
@@ -16,52 +16,23 @@ namespace Microsoft.PowerFx.Core.Texl.Builtins
     // Min(source:*, projection:n)
     // Max(source:*, projection:n)
     // Corresponding DAX functions: Min, MinA, MinX, Max, MaxA, MaxX
-    internal sealed class MinMaxTableFunction : StatisticalTableFunction
+    internal sealed class MinTableFunction : StatisticalTableFunction
```

Comment:
Why is this being split up and could it be extracted to a separate PR?

Reply:
It could be a separate PR.  It is being split up because after other changes to the class hierarchy and the removal of its own CheckTypes, there is little if any advantage to having them together.